rasa_nlu/server.py

```diff
@@ -58,6 +59,8 @@ def decorated(*args, **kwargs):
 
 def create_app(config, component_builder=None):
     rasa_nlu_app = Flask(__name__)
+    if 'cors_origins' in config:
```

Comment:
right - and that is probably a good habit. There is a reason though we don't do it for the configuration values (and the reason why the configuration isn't simply a dict but a separate class): it ensures that every configuration key we expect to be present is present and set to a default value. This keeps all the default values in one place and avoids the need to handle unset values at every point the configuration is accessed.